R/utils.R

```diff
@@ -457,22 +457,25 @@ valid_seq <- function(params) {
   }
 }
 
-resolve_prefix <- function(prefix, environment_var, base_pathname) {
+resolvePrefix <- function(prefix, environment_var, base_pathname) {
   if (!(is.null(prefix))) {
     assertthat::assert_that(is.character(prefix))
 
     return(prefix)
   } else {
+    # Check environment variables
     prefix_env <- Sys.getenv(environment_var)
-    if (prefix_env != "") {
+    env_base_pathname <- Sys.getenv("DASH_URL_BASE_PATHNAME")
+    app_name <- Sys.getenv("DASH_APP_NAME")
+    
+    else if (prefix_env != "")
       return(prefix_env)
-    } else {
-      env_base_pathname <- Sys.getenv("DASH_URL_BASE_PATHNAME")
-      if (env_base_pathname != "")
-        return(env_base_pathname)
-      else
-        return(base_pathname)
-    }
+    else if (app_name != "")
```

Comment:
Seems to be a typo here with the opening braces for the `else` statement that's preventing the package from installing.